public/404.html

```diff
@@ -62,59 +73,72 @@
-  <div class="collapse navbar-collapse">
-    <ul class="nav navbar-nav pull-right">
-      <li class='dropdown'><a href="#" class=dropdown-toggle" data-toggle="dropdown"><span class='fas fa-question-circle'></span> Help <span class='caret'></span></a>
-        <ul class="dropdown-menu dropdown-menu-right">
-          <li><a class='check-upload' href="https://github.com/broadinstitute/single_cell_portal/wiki" target='_blank'><span class='fas fa-info-circle fa-fw'></span> Documentation/Wiki</a></li>
-          <li><a href="https://github.com/broadinstitute/single_cell_portal/issues" target="_blank"><span class="fas fa-fw fa-exclamation-circle"></span> Issues</a></li>
-          <li><a href="https://github.com/broadinstitute/single_cell_portal_core" target="_blank"><span class="fab fa-fw fa-github"></span> Portal Source Code</a></li>
-        </ul>
-      </li>
-      <li class='disabled'><a href="#" id='login-nav'><span class='fas fa-sign-in fa-fw'></span> Login</a></li>
-    </ul>
-  </div>
-</nav>
-
-<div class="container-fluid" id="wrap">
-  <div class="row section-pad" id="main-body">
-    <div class="col-md-12">
-      <div class="bg-danger well well-lg" id="notice">
-        <h3 class="text-danger text-center">We're sorry, but the page you are looking for was not found.  Please try again.</h3>
-        <p class="lead text-center">Please <a href="https://github.com/broadinstitute/single_cell_portal/issues">report an issue</a> if this problem persists.</p>
+  <nav class="navbar fluid navbar-inverse navbar-fixed-top sc-navbar" role="navigation" id="single-cell-navbar">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a href="/single_cell" class="navbar-brand hidden-xs check-upload"><span class='fas fa-home'></span> Single Cell
+        Portal<sup class='text-danger'><small> BETA</small></sup></a>
+    </div>
+    <div class="collapse navbar-collapse">
+      <ul class="nav navbar-nav pull-right">
+        <li class='dropdown'><a href="#" class="dropdown-toggle" data-toggle="dropdown"><span
+              class='fas fa-question-circle'></span> Help <span class='caret'></span></a>
+          <ul class="dropdown-menu dropdown-menu-right">
+            <li><a class='check-upload' href="https://github.com/broadinstitute/single_cell_portal/wiki"
+                target='_blank'><span class='fas fa-info-circle fa-fw'></span> Documentation/Wiki</a></li>
+            <li><a href="https://github.com/broadinstitute/single_cell_portal/issues" target="_blank"><span
+                  class="fas fa-fw fa-exclamation-circle"></span> Issues</a></li>
+            <li><a href="https://github.com/broadinstitute/single_cell_portal_core" target="_blank"><span
```

Comment:
We still have the link to the issues in the dropdown for all of these files - probably we should change this to just be a direct `mailto` link that says 'Contact Us':

```
<li><a href="mailto:scp-support@broadinstitute.zendesk.com" target="_blank"><span
                  class="fas fa-fw fa-envelope"></span> Contact Us</a></li>
```

Reply:
updated in `d53230e`